beacon-chain/sync/subscriber_data_column_sidecar.go

```diff
@@ -52,3 +60,68 @@ func (s *Service) receiveDataColumnSidecar(ctx context.Context, sidecar blocks.V
 
 	return nil
 }
+
+// triggerGetBlobsV2ForDataColumnSidecar triggers getBlobsV2 retry when receiving a data column sidecar.
+// This function attempts to fetch the block and trigger the execution service's retry mechanism.
+func (s *Service) triggerGetBlobsV2ForDataColumnSidecar(ctx context.Context, blockRoot [32]byte) error {
+	// Check if service is properly configured
+	if s.cfg == nil || s.cfg.chain == nil || s.cfg.beaconDB == nil {
```

Comment:
I don't think we need to do this, you can check some other functions but some we just safely assume it's set